composer.json

```diff
@@ -131,7 +131,8 @@
         "patches": {
             "acquia/acquia-cms-starterkit": {
                 "ACMS-1988 - Update admin theme in starterkit to Gin": "https://github.com/acquia/acquia-cms-starterkit/compare/develop...feature/admin-gin-theme.patch",
-                "ACMS-3456: Update logic to enable theme.": "https://patch-diff.githubusercontent.com/raw/acquia/acquia-cms-starterkit/pull/132.patch"
+                "ACMS-3456: Update logic to enable theme.": "https://patch-diff.githubusercontent.com/raw/acquia/acquia-cms-starterkit/pull/132.patch",
+                "ACMS-4377: Add sitestudio_gin module in ACMS.": "https://patch-diff.githubusercontent.com/raw/acquia/acquia-cms-starterkit/pull/135.patch"
```

Comment:
@rajeshreeputra kindly remove this patch too.

Reply:
this will be removed once [PR](https://github.com/acquia/acquia-cms-starterkit/pull/135) is merged.

Reply:
its merged , we will see if the CI is passing after that or not